prow/config/config.go

```diff
@@ -368,6 +368,11 @@ type GitHubOptions struct {
 	// LinkURL is the url representation of LinkURLFromConfig. This variable should be used
 	// in all places internally.
 	LinkURL *url.URL
+
+	// APIEndpoints is a slice of strings that represents the api_endpoints config parameter.
```

Comment:
Don't need to explain in GoDoc to users that this implemented as a slice, the type is irrelevant. Should explain that we fallback on these on failure.